DuckDuckGo/AuthenticationViewController.swift

```diff
@@ -58,18 +58,37 @@ class AuthenticationViewController: UIViewController {
             onCouldNotAuthenticate()
         }
     }
+    
+    class Authenticator {
+        func canAuthenticate() -> Bool {
+            return true
+        }
+
+        func authenticate(reason: String) async throws -> Bool {
+            
+            return true
+        }
+    }
 
+// Added Async/Await
```

Comment:
We shouldn't add a comment here with what has changed - we use source control for that instead. 🙂